common/amount.c

```diff
@@ -370,6 +370,16 @@ bool amount_msat_to_u32(struct amount_msat msat, u32 *millisatoshis)
 	return true;
 }
 
+void amount_msat_from_u64(struct amount_msat *msat, u64 millisatoshis)
+{
+	msat->millisatoshis = millisatoshis;
+}
+
+void amount_msat_from_sat_u32(struct amount_msat *msat, u32 satoshis)
```

Comment:
Any particular reason you chose `u32` here? I think in most cases we use `u64` for satoshis as well.

Reply:
Because more than 4BTC as a min-value for channels seems crazy, but who am I to judge :D

... Changed it to 64bit.